ts/Core/ScrollbarDefaults.ts

```diff
@@ -118,6 +118,21 @@ const ScrollbarDefaults: ScrollbarOptions = {
      */
     minWidth: 6,
 
+    /**
+     * Defines the position of the scrollbar. By default, it is positioned
+     * on the opposite of the main axis (right side of the chart).
+     * However, in the case of RTL languages could be set to `false`
+     * which positions the scrollbar on the left.
+     *
+     * @sample stock/yaxis/scrollbar-opposite/
+     *         A scrollbar not on the opposite side
+     *
+     * @type      {boolean}
+     * @default   true
+     * @apioption scrollbar.opposite
+     */
+    opposite: true,
```

Comment:
That's a little tricky. Now all docs, [`xAxis`](http://localhost:9005/highstock/xAxis.opposite), [`yAxis`](http://localhost:9005/highstock/yAxis.opposite), and [`scrollbar`](http://localhost:9005/highstock/scrollbar.opposite) share this option, while only vertical Axis (yAxis in non-inverted and xAxis in the inverted chart) supports. I think this option should be removed from `scrollbar`, while `xAxis` and `yAxis` should mention when it works.

Reply:
Could you please elaborate because I don't fully get it?

> Now all docs, xAxis, yAxis,

The linked docs lead to `xAxis.opposit` which is a completely different parameter.

> while xAxis and yAxis should mention when it works.

Where this should be mentioned?

Reply:
> The linked docs lead to xAxis.opposit which is a completely different parameter.

My apologies! Should be `xAxis.scrollbar.opposite`, `xAxis.scrollbar.opposite` and `scrollbar.opposite`.

> Where this should be mentioned?

In the doclet, I think

Reply:
> I think this option should be removed from `scrollbar`, while `xAxis` and `yAxis` should mention when it works.

Now we have this option defined in `scrollbar` only :)